### ISSUE TITLE
Lesson 3 doesn't match Lesson 1

### ISSUE THREAD

**Issue by mcdickenson** [datacarpentry/cloud-genomics#11]
Lesson 1 uses ami-6516b30e. Lesson 3 uses ami-3c1c3454 which has a different organization and contents (and also appears not to be available in the AWS Community Marketplace). I think the solution is for Lesson 3 to be updated according to the organization and contents of ami-6516b30e. Are there plans to do this? (This may be the same issue that's described in #5.)

**Comment by mkcor** [comment]
@mcdickenson I would think so too, considering the information under "Amazon instance for workshop" found at http://www.datacarpentry.org/genomics-workshop/ (Sat Feb  6 22:46:34 EST 2016):
"""
All the software and data used in the workshop is on an Amazon AMI.

If you want to run your instance of the server used for this workshop, launch a t2.medium instance with AMI ami-6516b30e, available under "Community AMIs" in the Amazon EC2 Managemant Console.
"""

**Comment by tmorrell** [comment]
I wanted to bump this issue, because the presence of different images is confusing.  One potential problem changing the image names - lesson 3 uses the run SRR519926, and I'm not sure those files are in the ami-6516b30e image.  The cyverse/iplant instructions also don't provide access to the lesson 3 files (at least as far as I can figure out).

**Comment by devbioinfoguy** [comment]
I'm going to have to pull in Tracy & Jason on these, as I don't have access to the images. Clearly we'd like to ensure that there's one set of files being used. Let's see what we can do to rectify this problem.

-Bob

. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .

Bob Freeman, PhD
Director, Research Technology Operations @ Research Computing Services
Division of Research and Faculty Development    HARVARD | BUSINESS | SCHOOL
Baker Library & Bloomberg Center B96    25 Harvard Way, Boston, MA 02163

617.496.6612, phone    @DevBizInfoGuy    http://bit.ly/whatAboutBob

For more information –
RCS:                http://intranet.hbs.edu/dept/research/
Software:          http://intranet.hbs.edu/dept/research/software/
Rsrch Grid:        http://grid.rcs.hbs.org<http://grid.rcs.hbs.org/>
Training:            http://training.rcs.hbs.org<http://training.rcs.hbs.org/>

**Comment by devbioinfoguy** [comment]
I can probably help, let me take a look at this issue; I think I can get to it tomorrow.

- Jason

On Feb 27, 2017, at 5:03 PM, Freeman, Bob (Robert) <rfreeman@hbs.edu<mailto:rfreeman@hbs.edu>> wrote:

I'm going to have to pull in Tracy & Jason on these, as I don't have access to the images. Clearly we'd like to ensure that there's one set of files being used. Let's see what we can do to rectify this problem.

-Bob

. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .

Bob Freeman, PhD
Director, Research Technology Operations @ Research Computing Services
Division of Research and Faculty Development    HARVARD | BUSINESS | SCHOOL
Baker Library & Bloomberg Center B96    25 Harvard Way, Boston, MA 02163

617.496.6612, phone    @DevBizInfoGuy    http://bit.ly/whatAboutBob

For more information –
RCS:                http://intranet.hbs.edu/dept/research/
Software:          http://intranet.hbs.edu/dept/research/software/
Rsrch Grid:        http://grid.rcs.hbs.org<http://grid.rcs.hbs.org/>
Training:            http://training.rcs.hbs.org<http://training.rcs.hbs.org/>

**Comment by JasonJWilliamsNY** [comment]
Looking in the backend of AWS, the most recent image is ami-aab445c7 which I have just confirmed is public and searchable in community AMIs. Interestingly, the one mentioned in the 'Launching your own Instance' lesson is not to be found in our catalogue. 

To my knowledge, the AMI should not be mentioned except outside of that lesson, and I think when we did a PR we may have pulled in lessons that we don't us (We, meaning me and following this workshop as my personal reference for what lessons to use: http://www.datacarpentry.org/2016-05-26-NIH/) 

ami-aab445c7 is the latest instance

**Comment by tmorrell** [comment]
I sounds like the examples in lesson 3 are out of date.  Is it worth updating the content/does anyone use it?

**Comment by devbioinfoguy** [comment]
Yes, this lesson (#3) is likely going to go away (pending approval from Jason and Tracy) as most of this content is in other parts of the DC-genomics lessons, as was done as an temporary lesson early on in the DC-genomics evolution.

**Comment by tracykteal** [comment]
Yes, thank you. This is something very important to update. We'll be resolving this, this week.
